### PR TITLE
[traffic_light_classifier] support autoware auto msgs

### PR DESCRIPTION
**Pull request by taichiH**
## Description
port traffic light classifier

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [pull request guidelines][pull-request-guidelines]
- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [x] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: perception/traffic_light_recognition/traffic_light_classifier/include/traffic_light_classifier/nodelet.hpp [hunk 1]
```diff
@@ -66,20 +66,20 @@ class TrafficLightClassifierNodelet : public rclcpp::Node
...
-    sensor_msgs::msg::Image, autoware_perception_msgs::msg::TrafficLightRoiArray>
+    sensor_msgs::msg::Image, autoware_auto_perception_msgs::msg::TrafficLightRoiArray>
     ApproximateSyncPolicy;
   typedef message_filters::Synchronizer<ApproximateSyncPolicy> ApproximateSync;
   std::shared_ptr<ApproximateSync> approximate_sync_;
   bool is_approximate_sync_;
-  rclcpp::Publisher<autoware_perception_msgs::msg::TrafficLightStateArray>::SharedPtr
-    tl_states_pub_;
+  rclcpp::Publisher<autoware_auto_perception_msgs::msg::TrafficSignalArray>::SharedPtr
+    trafic_signal_array_pub_;
```
Comment (kmiya) [inline perception/traffic_light_recognition/traffic_light_classifier/include/traffic_light_classifier/nodelet.hpp:82]:
```suggestion
    traffic_signal_array_pub_;
```
Nit: Detected as typo. Please check the other warnings.

**Review by kmiya — CHANGES_REQUESTED** [review]

File: perception/traffic_light_recognition/traffic_light_classifier/include/traffic_light_classifier/nodelet.hpp (hunk 1 above)
Comment (kmiya) [reply perception/traffic_light_recognition/traffic_light_classifier/include/traffic_light_classifier/nodelet.hpp]: Fixed in 1509931.

**Review by yukkysaito — APPROVED** [review]

**Comment by taichiH** [comment]
@kmiya Approveお願いできないでしょうか？

**Review by kmiya — APPROVED** [review]
@taichiH お待たせしました :bow:

**Pull request merged**